web/core/src/jsMain/kotlin/org/jetbrains/compose/web/attributes/builders/TextAreaAttrsScope.kt

```diff
@@ -6,44 +6,46 @@
 package org.jetbrains.compose.web.attributes.builders
 
 import org.jetbrains.compose.web.attributes.*
+import org.jetbrains.compose.web.attributes.EventsListenerScope.Companion.BEFOREINPUT
+import org.jetbrains.compose.web.attributes.EventsListenerScope.Companion.INPUT
 import org.jetbrains.compose.web.events.SyntheticChangeEvent
 import org.jetbrains.compose.web.events.SyntheticSelectEvent
 import org.jetbrains.compose.web.events.SyntheticInputEvent
 import org.w3c.dom.HTMLTextAreaElement
 
-class TextAreaAttrsBuilder : AttrsBuilder<HTMLTextAreaElement>() {
+class TextAreaAttrsScope : AttrsScopeBuilder<HTMLTextAreaElement>() {
```

Comment:
can we have Deprecated typealiases for such cases too?

even though I think it's a rare case (that one have TestAreaAttrsBuilder instantiated in their code), it would be good if we try eliminate most cases which can break existing codebases.

Reply:
Added deprecation typealiases for the following:
* InputAttrsBuilder
* SelectAttrsBuilder
* TextAreaAttrsBuilder